index.js

```diff
@@ -128,7 +128,8 @@ module.exports = {
   lintTree: function(type, tree) {
     // Skip if useLintTree === false.
     if (this.options['ember-cli-qunit'] && this.options['ember-cli-qunit'].useLintTree === false) {
-      return tree;
+      // Fakes an empty broccoli tree
+      return { inputTree: tree, rebuild: function() { return []; } };
```

Comment:
That would be less awesome towards people not yet able to migrate to the latest version of Ember-CLI.